data/distro.yml

```diff
@@ -353,3 +353,36 @@
       </code></pre>
       <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
     </ol>
+
+- name: "Chrome OS"
+  logo: "chrome-os.svg"
+  info: >
+    <p>Flatpak applications can be installed on Chrome OS with the Crostini Linux compatibility layer. This is not available for all Chrome OS devices, so you should ensure your device is compatible before proceeding. A list of compatible devices is maintained <a href="https://www.reddit.com/r/Crostini/wiki/getstarted/crostini-enabled-devices">here</a>. Also note that prior to Version 69, Chrome OS required to be changed to the <strong>Developer channel</strong> before Crostini could be installed.</p>
```

Comment:
I don't think we need a note for pre-69 versions. Chrome OS is rolling, nothing supported is on old versions.

Reply:
sure.